FastSurferCNN/README.md

```diff
@@ -10,20 +10,25 @@ The network was trained with conformed images (UCHAR, 256x256x256, 1 mm voxels a
 The *FastSurferCNN* directory contains all the source code and modules needed to run the scripts. A list of python libraries used within the code can be found in __requirements.txt__. The main script is called __eval.py__ within which certain options can be selected and set via the command line:
```

Comment:
It was decided to call the script run_prediction.py and not eval.py anymore in vinn (more descriptive). So these commands need to be changed in the documentation for vinn